tasks/helm.yml

```diff
@@ -31,4 +31,4 @@ run:
       cd census-rm-kubernetes-dependencies-repo
       export HELM_TILLER_SILENT=true
       
-      ENV=${ENV} RABBITMQ_CONFIG_VALUES_FILE=${RABBITMQ_CONFIG_VALUES_FILE} ./setup-dependencies.sh
+      ENV=${ENV} RABBITMQ_CONFIG_VALUES_FILE=${RABBITMQ_CONFIG_VALUES_FILE} ./setup-dependencies.sh
```

Comment:
I don't think any pipeline still use this or prod-helm.yml so best deleted? They are still referenced in the fly pipelines jobs so that definitely needs clearing up.

Reply:
The idea is to keep them for now because we don't want to mess with any of the pipeline jobs in other environments. Once this goes in it'll break all the pipelines if theres still releases needing to go in. Once we want to release to blacklodge, greylodge etc. we can update the `rabbitmq-file` variable in concourse-pipelines.yml to point to the new rabbit operator and then delete the old tasks

Reply:
So this file should have no changes? Looks like the only change is the trailing new lines have been removed, may be better to revert it and remove it from this PR completely.

Reply:
It's gone! 👍